src/blocks/dropdown/Dropdown.types.ts

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from 'react';
+
+export type DropdownProps = {
+  // This will be content upon clicking on which the dropdown overlay will open
+  children?: (isOpen: boolean) => ReactNode;
```

Comment:
```suggestion
  children: (isOpen: boolean) => ReactNode;
```
children should be mandatory for dropdown